### DISCUSSION TITLE
Terminal and font transparency

### DISCUSSION THREAD

**Discussion by g4xx** [excalith/excalith-start-page#57] [Ideas]
It would be nice to separate terminal transparency from font transparency. Currently using opacity parameter really hurts legibility of text if set too low. This of course could be two separate parameters in config.

**Comment by excalith** [comment]
That is something bothered me since I added that feature. It needs some refactoring under the hood to prevent texts being effected by the parent container. However, I will check this out once I manage to find a proper way for the docker image. Converting this to a feature request, thanks!
